src/libsync/syncengine.cpp

```diff
@@ -831,9 +831,10 @@ void SyncEngine::slotInsufficientLocalStorage()
 
 void SyncEngine::slotInsufficientRemoteStorage()
 {
-    auto msg = tr("There is insufficient space available on the server for some uploads.");
-    if (_uniqueErrors.contains(msg))
+    const QString msg = tr("Space quota exceeded. Please contact the Administrator of this space.");
```

Comment:
a space has no admin 🙈 only a manager. that's the harder to explain part... space quota can be changed by: a) global admin, b) manager of the space.

Reply:
Contact your space manager could also lead to confusion....